packages/bem-react-scripts/scripts/build.js

```diff
@@ -68,132 +68,18 @@ function build(previousFileSizes) {
 
   let compiler;
   try {
-    compiler = webpack(config);
+    compiler = parallelWebpack.run(
+        configPath,
+        {
+            watch: false,
+            maxRetries: 1,
+            stats: true // defaults to false
```

Comment:
Better to use ENV for this value

Reply:
Prob better to use params like in parallel-webpack: https://github.com/trivago/parallel-webpack#specifying-retry-limits